posthog/temporal/data_imports/sources/doit/source.py

```diff
@@ -0,0 +1,61 @@
+from typing import cast
+from posthog.schema import (
+    ExternalDataSourceType,
+    SourceConfig,
+    SourceFieldInputConfig,
+    Type4,
+)
+from posthog.temporal.data_imports.sources.doit.doit import doit_source, doit_list_reports, DOIT_INCREMENTAL_FIELDS
```

Comment:
NIT: opportunity to fix up the naming here. This is the source file but then the "thing" we pass to the pipeline is a source too. DotItSource and doit_source

Reply:
Yeah, naming things is hard. Will give this some more thought